src/main/java/com/bart/scorebetlive442/model/League.java

```diff
@@ -0,0 +1,21 @@
+package com.bart.scorebetlive442.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import java.util.Set;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+public class League {
+
+    private Long id;
+    private String name;
+    private String country;
+    private Set<Team> teams;
+    private String level;
```

Comment:
co ma znaczyć level? Poziom rozgrywek ? A to nie liczba, typu Ekstraklasa - 1, I Liga - 2,.
teoretycznie jest to zbędne "nazwa ligi może determinować jej poziom"